src/System.Net.HttpListener/tests/SimpleHttpTests.cs

```diff
@@ -75,29 +79,33 @@ public void BasicTest_StartAbortClose_NoException()
             }
         }
 
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
+        [Fact]
+        [ActiveIssue(20009, TargetFrameworkMonikers.Uap)]
         public void BasicTest_StopNoStart_NoException()
         {
             HttpListener listener = new HttpListener();
             listener.Stop();
             listener.Close();
         }
 
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
+        [Fact]
+        [ActiveIssue(20009, TargetFrameworkMonikers.Uap)]
```

Comment:
>should all these actually be #17462

Both issues are "epics".  But #17462 is the sub-epic just for UWP HttpListener work. So it should be 17462